recipes/backward-cpp/all/conanfile.py

```diff
@@ -0,0 +1,152 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class BackwardCppConan(ConanFile):
+    name = "backward-cpp"
+    description = "A beautiful stack trace pretty printer for C++"
+    homepage = "https://github.com/bombela/backward-cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "backward-cpp", "stack-trace")
+    license = "MIT"
+    exports_sources = [ "CMakeLists.txt", "patches/backward-cpp-*.patch" ]
+    generators = "cmake"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+       "stack_walking_unwind": [True, False],
+       "stack_walking_backtrace": [True, False],
+       "stack_details_auto_detect": [False], # dont let backtrace auto decide
```

Comment:
Remove this option in the recipe and just hardcode the False value in the Cmake definition down below